docs/queries/queries.md

```diff
@@ -270,7 +300,7 @@ profiles = await Profile.query.select_related("user").filter(email__icontains="f
 
 ### All
 
-Returns all the instances.
+Copy the query. When providing True, instead copying the cache is cleared and the same query returned.
```

Comment:
This is way too technical. The all must return all the results. If it's cached or not, the user doesn't care neither should. Revert this please. Keep it simple as it was

Reply:
hope my new doc is ok